caddyhttp/httpserver/server.go

```diff
@@ -363,6 +363,8 @@ func (s *Server) OnStartupComplete() {
 			output += " (only accessible on this machine)"
 		}
 		fmt.Println(output)
+		//Also print to -log.
+		log.Println(output)
```

Comment:
I'm not sure I understand.  By my basic testing if I run `caddy -quiet` I get no output. 

If I run `caddy -log stdout -quiet` I get the Startup and SIGINT log but not the output of line 367

Am I missing something?